apps/pie-microsite/src/_includes/content-page.njk

```diff
@@ -0,0 +1,25 @@
+<div class="c-contentPage">
+    <div class="c-contentPage-header">
+        <h1 class="alpha--standout">
+            {{ title }}
+        </h1>
+        <p class="c-contentPage-header-subheading">
+            {{ description }}
+        </p>
+    </div>
+
+    {% if navKey %}
+        {% include "page-tabs.njk" %}
+    {% endif %}
+
+    {% if content %}
+        <div class="c-contentPage-main">
+            {{ content | safe }}
+        </div>
+    {% else %}
+        <div class="c-contentPage-comingSoon">
```

Comment:
Slightly NIT-ty, but I'd be tempted to make this a bit less specific, so it's more reusable in future.

```suggestion
        <div class="c-contentPage-main">
            {{ content | safe }}
        </div>
    {% else %}
        <div class="c-contentPage-main c-contentPage-main--fullWidth">
```

fullWidth may not be the best name, but just having the name more around what the styles are aiming to achieve.

And then you can just apply the background to `c-contentPage-main`, rather than having two separate selectors.

Also, separate comment. If this is the main – would it be worth using a `<main>` element?